Mithril update 2021-11-01 (#281)
input-output-hk/cardano-updates: blog/2023-11-01-mithril.md

---
title: Mithril Team Update
slug: 2023-11-01-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

This week, the Mithril team kept working on decentralizing the Mithril networks with the peer-to-peer (P2P) networking proof of concept. They progressed with the adaptation of the Mithril client as a library, and worked on its full WASM compatibility in a proof of concept. The team kept working on the deterministic computation of the transactions from the immutable files, and fixed some unreachability issue on the `release-mainnet` aggregator.

> Finally, they implemented new Docker images that build faster the Mithril nodes of the `devnet`, and they fixed some regression on the Mithril client CLI.

Reviewer: ```suggestion
Finally, they implemented new Docker images that build the Mithril nodes on the 'devnet' faster and fixed some regressions in the Mithril client CLI.
```
